previews/primer/alpha/nav_list_preview/trailing_action.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render(Primer::Alpha::NavList.new) do |list| %>
+  <% list.with_group do |group| %>
+    <%= group.with_heading(title: "Shopping list") %>
+    <% group.with_item(label: "Bread", href: "/list/1") do |item| %>
+      <%= item.with_trailing_action(show_on_hover: true, icon: :plus, aria: { label: "Click for alert" }, name: "bread_button") %>
```

Comment:
This is super nitpicky but can we change the phrasing of the `aria-label`? Just because not everyone will be clicking on the item, they may be pressing the enter key! I know this is just for an example but it might prevent someone from trying to do this in the Real World™️ 
```suggestion
      <%= item.with_trailing_action(show_on_hover: true, icon: :plus, aria: { label: "Activate alert" }, name: "bread_button") %>
```

Reply:
Love the ™ lol